packages/api-client/src/main/tcp/ReconnectingWebsocket.ts

```diff
@@ -118,6 +139,30 @@ export class ReconnectingWebsocket {
     }
   }
 
+  private startPinging(): void {
+    this.stopPinging();
```

Comment:
Stops pinging after the start?

Reply:
No, that's stopping before starting it. There are cases where we had 2 pingers running, that's why the failed ping check was actually failing.